ed_sensor_integration/include/ed/kinect/fitter.h

```diff
@@ -103,17 +105,21 @@ class Fitter
     /**
      * @brief Fitter constructor
      * @param nr_data_points nr_data_points for the beam model
+     * @param fx focal length for the beam model
      */
-    Fitter(uint nr_data_points = 200);  // TODO: remove hard-coded values
+    Fitter(uint nr_data_points = 200, float fx = 100);
```

Comment:
The stupid answer is: if it doesn't the CI tests break. So arguably I should just fix the testcase rather than use these hard-coded values.
But this format for the Fitter should still exist. as nr of points and focal length is a valid definition of the fitter model